packages/headless/src/state/search-app-state.ts

```diff
@@ -50,4 +51,5 @@ export type SearchAppState = SearchParametersState &
   SearchSection &
   DidYouMeanSection &
   FieldsSection &
-  FacetOrderSection & {history: StateWithHistory<HistoryState>};
+  FacetOrderSection &
+  VersionSection & {history: StateWithHistory<HistoryState>};
```

Comment:
Why is `VersionSection` added to SearchParametersState ? 🤔 

That's not a search parameter.

Reply:
GitHub collapsed part of the code, making it look like it's part of SearchParametersState.

This is actually part of SearchAppState.